Assets/GameAssets/Boss/BossScripts/LookAtMoveDirection.cs

```diff
@@ -0,0 +1,38 @@
+using UnityEngine;
+
+public class LookAtMoveDirection : MonoBehaviour
+{
+    private Vector3 lastPosition; // Store the last frame's position
+    [SerializeField] private float rotationSpeed = 10f; // Speed at which the object rotates
+
+    /// <summary>
+    /// sets first previous position
+    /// </summary>
+    private void Start()
+    {
+        // Initialize the last position
+        lastPosition = transform.position;
+    }
+
+    /// <summary>
+    /// changes the rotation of the the object to look at the direction it is moving toward
+    /// </summary>
+    private void Update()
+    {
+        // Calculate the velocity (difference in position over time)
+        Vector3 velocity = (transform.position - lastPosition) / Time.deltaTime;
+
+        // Update the last position for the next frame
+        lastPosition = transform.position;
```

Comment:
Maybe we could store a Transform so we don't have to update it every frame? No clue if that would be more efficient tho